src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -212,49 +213,184 @@ private void testCreateInstructorAction() throws Exception {
 
         feedbackEditPage.clickEditSessionButton();
 
         FeedbackSessionAttributes feedbackSession =
+                getFeedbackSessionWithRetry(demoCourseId, "First team feedback session");
+        feedbackEditPage.editFeedbackSession(
+                feedbackSession.getStartTime(), feedbackSession.getEndTime(),
+                new Text("updated instructions"), feedbackSession.getGracePeriod());
+        feedbackEditPage.reloadPage();
+        feedbackSession = getFeedbackSessionWithRetry(demoCourseId, "First team feedback session");
+        assertEquals("<p>updated instructions</p>", feedbackSession.getInstructionsString());
+
+        ______TS("new instructor can edit Second team feedback session of sample course");
+        instructorHomePage.loadInstructorHomeTab();
+        feedbackEditPage =
+                instructorHomePage.clickFeedbackSessionEditLink(demoCourseId, "Second team feedback session");
+
+        feedbackEditPage.clickEditSessionButton();
+
+        feedbackSession = getFeedbackSessionWithRetry(demoCourseId, "Second team feedback session");
+        feedbackEditPage.editFeedbackSession(
+                feedbackSession.getStartTime(), feedbackSession.getEndTime(),
+                new Text("updated instructions"), feedbackSession.getGracePeriod());
+        feedbackEditPage.reloadPage();
+        feedbackSession =
                 getFeedbackSessionWithRetry(demoCourseId, "Second team feedback session");
-        feedbackEditPage.editFeedbackSession(feedbackSession.getStartTime(),
-                                             feedbackSession.getEndTime(),
-                                             new Text("updated instructions"),
-                                             feedbackSession.getGracePeriod());
+        assertEquals("<p>updated instructions</p>", feedbackSession.getInstructionsString());
+
+        ______TS("new instructor can edit Third feedback session of sample course");
+        instructorHomePage.loadInstructorHomeTab();
+        feedbackEditPage =
+                instructorHomePage.clickFeedbackSessionEditLink(demoCourseId, "Session with different question types");
+
+        feedbackEditPage.clickEditSessionButton();
+
+        feedbackSession = getFeedbackSessionWithRetry(demoCourseId, "Session with different question types");
+        feedbackEditPage.editFeedbackSession(
+                feedbackSession.getStartTime(), feedbackSession.getEndTime(),
+                new Text("updated instructions"), feedbackSession.getGracePeriod());
         feedbackEditPage.reloadPage();
-        instructorHomePage.verifyHtmlMainContentWithReloadRetry("/newlyJoinedInstructorFeedbackSessionSuccessEdited.html");
+        feedbackSession = getFeedbackSessionWithRetry(demoCourseId, "Session with different question types");
+        assertEquals("<p>updated instructions</p>", feedbackSession.getInstructionsString());
```

Comment:
There is no need to check if instructor can _edit_ each session. Rather, we are checking to see if the data of the sample course is as expected.